src/containerapp/azext_containerapp/_params.py

```diff
@@ -66,16 +67,29 @@ def load_arguments(self, _):
 
     # Telemetry 
     with self.argument_context('containerapp env telemetry data-dog set') as c:
-        c.argument('site', options_list=['--site'], help='Specify the data dog site')
-        c.argument('key', options_list=['--key'], help='Specify the data dog api key')
+        c.argument('name', id_part=None)
```

Comment:
you can add  it once in `containerapp env telemetry` group level and it will take effect for all commands inside